x-pack/plugins/log_explorer/public/components/virtual_columns/column_tooltips/field_with_token.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiText, EuiToken } from '@elastic/eui';
+import React from 'react';
+import { css } from '@emotion/react';
+import { euiThemeVars } from '@kbn/ui-theme';
+
+export const FieldWithToken = ({
+  field,
+  iconType = 'tokenKeyword',
+}: {
+  field: string;
+  iconType?: string;
+}) => {
+  const spacingXsCss = css`
+    margin-bottom: ${euiThemeVars.euiSizeXS};
+  `;
```

Comment:
**nit**: move this out of the rendering, it doesn't have anything depending on the component :)